docs/components/Colors.css

```diff
@@ -0,0 +1,51 @@
+:root {
+  --color-width: 177px;
+  --color-height: 60px;
+  --hex-font-size: 11px;
+  --border-color: #f0f0f0;
+  --color-text-white: #fefefe;
+  --color-text-gray: var(--accent-dark);
+}
+
+.Colors {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 20px var(--space-medium);
+}
+
+.Color {
+  width: var(--color-width);
+  height: var(--color-height);
+  display: inline-block;
+  margin-right: 20px var(--space-medium);
```

Comment:
I think we can achieve what you're doing here in more simple terms:

```css
.Colors {
  display: grid;
  grid-template-columns: repeat(4, 1fr);
  gap: var(--space-small);
}

.Color__bg {
  height: 3rem;
  position: relative;
}
```

The other advantage of removing the set fixed/widths is that the color tiles will always fill the container, and we can apply media queries to shrink down the column count when appropriate:

```css
@media(max-width: 38rem) { 
  .Colors {
    grid-template-columns: repeat(3, 1fr);
  }
}

@media(max-width: 36rem) { 
  .Colors {
    grid-template-columns: repeat(2, 1fr);
  }
}

...

```

The above code is an example, you will have to play around with the media queries to see what feels right here.